giraffe/src/utils/fromFlux.ts

```diff
@@ -378,7 +383,7 @@ export const fastFromFlux = (fluxCSV: string): FromFluxResult => {
   This function splits up a CSV response into these individual CSV files.
   See https://github.com/influxdata/flux/blob/master/docs/SPEC.md#multiple-tables.
 */
-const splitChunks = (fluxCSV: string): string[] => {
+export const parseChunks = (fluxCSV: string): string[] => {
```

Comment:
why the rename if this is splitting chunks?

Reply:
this function had a duplicate so I kept the naming of the other duplicate version.

Reply:
k, so is `parseChunks` used anywhere else but the `fromFlux` function?

Reply:
🤔 thanks for pointing me to that. Looks like we should clean that up in a follow-up PR unrelated to this work

Reply:
How do you mean clean it up? Like rename it later?

Reply:
Oh no, We're in the process of removing that function altogether since it's one of the most resource-intensive operations in parsing out CSV results.

Reply:
Ahhh, i see. So just to be clear, that is good for now but we can just keep in mind that it needs cleanup with a better way to parse, but in a future PR?